### PR TITLE
Greenpack

### PR DESCRIPTION
**Pull request by kochamale123**
added green weapons nd stuff

### CODE REVIEW

File: texturepacks/greenpack/_meta.json
Comment (itzTheMeow) [inline texturepacks/greenpack/_meta.json:1]: make the description a little more descriptive

**Review by itzTheMeow — CHANGES_REQUESTED** [review]
description change, everything else looks good

File: texturepacks/greenpack/_meta.json
Comment (kochamale123) [reply texturepacks/greenpack/_meta.json:1]: so i should do it all over again just changing the description right

Comment (kochamale123) [reply texturepacks/greenpack/_meta.json:1]: done

**Review by itzTheMeow — APPROVED** [review]
cool

**Pull request merged**